libr/bin/bin.c

```diff
@@ -1285,6 +1285,20 @@ R_IPI void r_bin_class_free(RBinClass *c) {
 	free (c);
 }
 
+static RBinClass *class_get(RBinFile *binfile, const char *name) {
+	r_return_val_if_fail (binfile && binfile->o && name, NULL);
+
+	RBinClass *c;
+	RListIter *iter;
+	RList *list = binfile->o->classes;
+	r_list_foreach (list, iter, c) {
```

Comment:
Add a warning message because this is o(n) and we shohld use a kv to resolve this faster